requirements.txt

```
@@ -1,3 +1,4 @@
 terminaltables
 requests
 PyYAML
+packaging==23.0
```

```suggestion
packaging>=23.0
```

I think >= might be better since someone might want to use a specific version of `packaging` in the future.